src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -166,6 +166,414 @@ E. Click the *Tasks* tab, select an *Preview_Results* task and click on the *Pre
+construct tasks and workflows to execute them on the fly.
+
+=== Installation
+
+==== Requirements:
+
+Python 2 or 3
+
+==== Using PyPi
+
+* open a terminal
+
+* install the proactive jupyter kernel
+
+```bash
+$ pip install proactive proactive-jupyter-kernel --upgrade
+$ python -m proactive-jupyter-kernel.install
+```
+
+==== Using source code
+
+* open a terminal
+
+* clone the repository on your local machine:
+
+```bash
+$ git clone git@github.com:ow2-proactive/proactive-jupyter-kernel.git
+```
+
+* install the proactive jupyter kernel:
+
+```bash
+$ pip install proactive-jupyter-kernel/
+$ python -m proactive-jupyter-kernel.install
+```
+
+=== Platform
+
+You can use any jupyter platform.
+We recommend the use of jupyter lab. To launch it from your terminal after having installed it:
+
+```bash
+$ jupyter lab
+```
+or in daemon mode:
+```bash
+$ nohup jupyter lab &>/dev/null &
+```
+
+When opened, click on the Proactive icon to open a notebook based on the proactive kernel.
+
+=== Connection
+
+==== Using connect()
+
+If you are trying proactive for the first time, please sign up on https://try.activeeon.com/signup.html[try platform].
+Once you receive your login and password, connect using the `#%connect()` pragma:
+
+```python
+#%connect(login=YOUR_LOGIN, password=YOUR_PASSWORD)
+```
+
+To connect to another host, please use the later pragma this way:
+
+```python
+#%connect(host=YOUR_HOST, port=YOUR_PORT, login=YOUR_LOGIN, password=YOUR_PASSWORD)
+```
+
+==== Using config file:
+
+For automatic sign in, create a file named *proactive_config.ini* in your notebook location.
+
+Fill your configuration file according to the format:
+
+```ini
+[proactive_server]
+host=YOUR_HOST
+port=YOUR_PORT
+[user]
+login=YOUR_LOGIN
+password=YOUR_PASSWORD
+```
+
+Save your file changes and restart the proactive kernel.
+
+You can also force the current Kernel to connect using any .ini config file through the `#%connect()` pragma:
+
+```python
+#%connect(path=PATH_TO/YOUR_CONFIG_FILE.ini)
+```
+
+(for more information about this format please check
+https://docs.python.org/3/library/configparser.html[configParser])
+
+=== Usage
+
+===== Creating a Python task
+
+To create a task, please use the pragma `#%task()` followed by the task implementation script wrote into a notebook block code.
+To use this pragma, at least, a task name has to be provided. Example:
+
+```python
+#%task(name=myTask)
+print('Hello world')
+```
+
+General usage:
+
+```python
+#%task(name=TASK_NAME, [language=SCRIPT_LANGUAGE], [dep=[TASK_NAME1,TASK_NAME2,...]], [generic_info=[(KEY1,VAL1), (KEY2,VALUE2),...]], [export=[VAR_NAME1,VAR_NAME2,...]], [import=[VAR_NAME1,VAR_NAME2,...]], [path=IMPLEMENTATION_FILE_PATH])\n'
+```
+
+As seen in the general usage, users can also provide more information about the task by using the pragma's options:
+
+====== Language
+
+The `language` parameter is needed when the task script is not written in native Python, the default language.
```

Comment:
(maybe highlight this point about default language because it's an important one ), the default language => . If not provided, Python will be selected as the default language.